source/gamefunctions_debug.bmx

```diff
@@ -1470,6 +1443,31 @@ Type TDebugScreen
 		'handled
 		sender.clicked = False
 		sender.selected = False
+
+
+		Function printLog:Int(heading:String, _function:String[](playerID:Int, day:Int), logFileName:String)
+			Local playerIDs:Int[] = [1,2,3,4]
+			Local logFiles:TStream[4]
+			For Local playerID:Int = EachIn playerIDs
+				logFiles[playerID-1] =  WriteStream("utf8::" + "logfiles/log."+logFileName+"_"+playerID+".txt")
```

Comment:
Hier ueberpruefen, ob WriteStream erfolgreich war - also `logFiles[playerID-1] <> Null`.

nur wenn logfiles... existiert, sollte der Rest gemacht werden. Oder es sollte iwie drauf reagiert werden, dass die Datei halt nicht angelegt werden konnte (bereits in einem Editor geoeffnet, Schreibschutz, ...)

Reply:
Ich habe die Null-Prüfung eingebaut. Mehr sollte doch eigentlich nicht nötig sein. Es ist ja keine normalspielrelevante Funktion.